data/import/common/common/glossary.csv

```diff
@@ -1343,6 +1345,8 @@ product.filter.price-default-eur-net_mode,Preis,de_DE
 product.filter.price-default-eur-net_mode,Price range,en_US
 product.filter.price-default-eur-gross_mode,Preis,de_DE
 product.filter.price-default-eur-gross_mode,Price range,en_US
+product.filter.price-default-usd-net_mode,Price range,en_US
+product.filter.price-default-usd-gross_mode,Price range,en_US
```

Comment:
Don't we need a DE translation?

Reply:
this is just an integration
if we need it, then it should be inside the main PR
https://github.com/spryker/suite-nonsplit/pull/8613/files#diff-591493093b412548fb171d04b9e15b0b4c2543abdd5b6a41948c480b94a6db6c